APE9.rst

```diff
@@ -0,0 +1,111 @@
+A General Timeseries Class for Astropy
+-------------------------------------
+
+author: Stuart Mumford
+
+date-created: 2016 March 24
+
+date-last-revised: 2016 March 24
+
+type: Standard Track
+
+status: Discussion
+
+
+Abstract
+--------
+
+The goal of a timeseries object in astropy is to provide a core set of
+functionality that can be shared between astropy, affiliated packages and the
+wider community.
+
+The scope of this timeseries object is designed to remain focused on the core
```

Comment:
Should we mention performance? It should work well and quickly for large data sets (million entries) as well as small.